docs/proposals/parameterized-config-management-plugins.md

```diff
@@ -211,15 +209,17 @@ spec:
   parameters:
     # The declarative announcement follows the parameters announcement schema. This is where a parameter description
     # should go if it applies to all apps for this CMP.
-    announcement:
-      - name: example
-    # The (optional) generated announcement is combined with the declarative announcement (if present).
+    - name: values-file
+      title: Values File
+      tooltip: Path of a Helm values file to apply to the chart.
+  # The (optional) generated announcement is combined with the declarative announcement (if present).
+  announcement:
```

Comment:
isn't this related to the dynamic parameters? If so, how about changing: `s/announcement/dynamicParameters`?

Reply:
yep, I like that.